README.md

```diff
@@ -19,99 +51,105 @@ The following matrix lists supported versions of Kong for every release of Kong
 
 ## Get started
 
-Please check the [deployment documentation][1]
-
-## How it works?
-
-Using a standard [Kubernetes deployment][10], the Ingress controller runs multiple containers in a single pod. This allows us to define in a unit containing an [initContainer][11] to run Kong migrations, one container for the Kong admin API in control-plane mode and one container for the ingress controller itself. With this approach we simplify the deploy of the required components without user intervention.
-Once the deployment passes the readiness and liveness probes it means the Kong migrations ran and ingress controller can communicate with the Kong admin API.
-
-In a different deployment we run Kong in data-plane mode. This means the Kong instances only expose the proxy ports.
-By using this approach we can deploy and scale the data-plane with the requirements of your applications, i.e. using a daemonset, a deployment with affinity rules, etc.
+You can deploy Kong Ingress Controller on any
+Kubernetes cluster which Service of `type: LoadBalancer`.
 
-The next diagram shows how the components interact:
+You can use
+[Minikube](https://kubernetes.io/docs/setup/minikube/)
+on your local machine or use
+a hosted k8s service like
+[GKE](https://cloud.google.com/kubernetes-engine/).
```

Comment:
Now that it is GA, I'd suggest you also mention Docker Desktop https://www.docker.com/products/docker-desktop - it makes it fast and easy to get started with Kubernetes.

Reply:
This is new!
We've not tested or worked with it yet, so we can't include it yet as we won't be able to solve support issues with it.
Once we're familiar and have verified that it works well with our Ingress Controller, we sure can include it!